django-server/server/server/settings.py

```diff
@@ -101,6 +102,10 @@
 def select_database():
     if env.bool('DB_DEPLOY', default=False):
         return env.db()
+    if env.bool('USE_POSTGRES', default=False):
+        print("Using postgres")
```

Comment:
remove oall debug prints from release branch or create a module for debugind that is configured via environment variable and we will set it to false on serevr